src/CrystallineSociety/Client/Shared/Shared/Header.razor.scss

```diff
@@ -2,59 +2,6 @@
 @import '../Styles/abstracts/_colors.scss';
 @import '../Styles/abstracts/_functions.scss';
 
-.header {
-    position: fixed;
-    top: 0;
-    z-index: 100;
-    display: flex;
-    flex-flow: row nowrap;
-    justify-content: space-between;
-    align-items: center;
-    width: 100%;
-    height: rem(48px);
-    padding: 0 rem(32px);
-    box-sizing: border-box;
-    background-color: $Gray5;
-
-    @media all and (max-width: #{em(370px)}) {
-        padding: 0 rem(16px);
-    }
-}
-
-.logo-lnk {
-    width: rem(24px);
-    min-width: rem(24px);
-    height: rem(24px);
-    border-radius: rem(4px);
-    background-image: url('images/bit-logo.svg');
-    background-position: center;
-    background-repeat: no-repeat;
-    background-size: contain;
-    cursor: pointer;
-}
-
-.header-contact {
-    display: flex;
-    flex-flow: row nowrap;
-    justify-content: flex-start;
-    align-items: center;
-}
-
-::deep .header-item {
-    margin-left: rem(12px);
-}
-
-::deep .header-menu {
-    display: none;
-    margin-right: rem(12px);
-
-    span {
-        i {
-            color: $PrimaryColor;
-        }
-    }
-
-    @include lt-lg {
-        display: block;
-    }
+.user-box {
+    text-align: right;
```

Comment:
We have this attribute class name on Bootstrap

Reply:
done